rocAL/include/meta_data/bounding_box_graph.h

```diff
@@ -31,8 +31,8 @@ typedef  struct { float xc; float yc; float w; float h; } BoundingBoxCord_xcycwh
 class BoundingBoxGraph : public MetaDataGraph {
    public:
     void process(pMetaDataBatch input_meta_data, pMetaDataBatch output_meta_data) override;
-    void update_meta_data(pMetaDataBatch meta_data, decoded_image_info decode_image_info) override;
-    void update_random_bbox_meta_data(pMetaDataBatch input_meta_data, pMetaDataBatch output_meta_data, decoded_image_info decoded_image_info, crop_image_info crop_image_info) override;
+    void update_meta_data(pMetaDataBatch meta_data, decoded_sample_info decode_image_info) override;
```

Comment:
Is the meta_data always corresponds to image as well? Also I am noticing the struct name and variable name is same here. Google coding style required camelcase for struct and snake_case for variable names

Reply:
bounding_box_graph file is meant to handle images meta-data only.